jsonnet/rules.libsonnet

```
@@ -21,6 +21,36 @@ local droppedKsmLabels = 'endpoint, instance, job, pod, service';
               severity: 'warning',
             },
           },
+          {
+            expr: |||
+              count by(node,workload,namespace)
+                (
+                  kube_pod_info{node!=""}
+                  * on(namespace,pod) group_left()
+                  (
+                    kube_pod_spec_volumes_persistentvolumeclaims_info
+                  )
+                  * on(namespace,pod) group_left(workload)
+                  (
+                    namespace_workload_pod:kube_pod_owner:relabel
+                    * on(namespace,workload,workload_type) group_left()
+                    (
+                      count without(pod) (namespace_workload_pod:kube_pod_owner:relabel) > 1
+                    )
+                  )
+                ) > 1
```

```suggestion
              count by(node,workload,namespace)
                (
                  kube_pod_info{node!=""}
                  * on(namespace,pod) group_left()
                  (
                    kube_pod_spec_volumes_persistentvolumeclaims_info
                  * on(namespace,pod) group_left(workload)
                  (
                    namespace_workload_pod:kube_pod_owner:relabel
                    * on(namespace,workload,workload_type) group_left()
                    (
                      count without(pod) (namespace_workload_pod:kube_pod_owner:relabel) > 1
                    )
                  )
                 )
                ) > 1
```

I _think_ this might fix the `found duplicate series for the match group` error.

---

phew..that is a simple change, even if it doesn't look that way.
It just move a closing parenthesis from line 32 to line 38,  ensuring that the right operand of the first multiplication is calculated first.

---

Indeed while converting the query to jsonnet, I removed one level of indentation, good catch!